### ISSUE TITLE
Multi-arch support

### ISSUE THREAD

**Issue by DeepMarks** [j6k4m8/goosepaper#43]
Hi, thank you very much for creating Goosepaper, I love reading them in the morning!

Suggestion: I noticed that on Linux the image is specifically designed for an amd architecture so it rejects my arm architecture:

WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm/v7) and no specific platform was requested
standard_init_linux.go:219: exec user process caused: exec format error

Would be great if you could push an image with multi-arch support, preferably for use in docker. Thanks already in advance.

Cheers!

**Comment by sedennial** [comment]
I have the same issue since I'm running mine on a Pi (alongside one of my Plex servers). I know zero about Docker so I ended up doing the standalone install.

**Comment by DeepMarks** [comment]
> I have the same issue since I'm running mine on a Pi (alongside one of my Plex servers). I know zero about Docker so I ended up doing the standalone install.

I have the exact same setup, is the standalone working for you?

**Comment by sedennial** [comment]
> I have the exact same setup, is the standalone working for you?

Yep. I just followed the walkthrough though I did have an issue with weasyprint at first but it was easily resolved. I just don't remember what it was. :)

**Comment by DeepMarks** [comment]
> > I have the exact same setup, is the standalone working for you?
> 
> Yep. I just followed the walkthrough though I did have an issue with weasyprint at first but it was easily resolved. I just don't remember what it was. :)

Great to know thanks, did you set up an instance that sends the paper every few hours/days or are you doing it manually? Would be interested!

**Comment by sedennial** [comment]
I set up cronjobs for various feeds. I created several custom feeds from rssmix also. I have some daily and a couple that are at 0500, 1400, and 2000 hours.

I'm trying to get panadoc and wget to work for authenticated html pages. If I can get those working I intend to try and add support for that. I have some unclassified threat Intel pages I'd love to get on my RM.

**Comment by DeepMarks** [comment]
> I set up cronjobs for various feeds. I created several custom feeds from rssmix also. I have some daily and a couple that are at 0500, 1400, and 2000 hours.
> 
> I'm trying to get panadoc and wget to work for authenticated html pages. If I can get those working I intend to try and add support for that. I have some unclassified threat Intel pages I'd love to get on my RM.

Thanks for sharing, I was looking into cron as well and it seems to be a great option :)

**Comment by j6k4m8** [comment]
You wonderful wonderful people! I am so grateful for your insights here :) 

I am working on a CI/CD setup to automatically regenerate docker images; right now it relies on the Docker automated push/pull, which is no good. I will make this a priority! Any thoughts/help is of course very welcome!

If you have any docs you'd be willing to contribute, I'd love to incorporate them into the repo for others to follow along too.

As an aside, I've been using a GitHub repo with an Action set with a time trigger with a lot of success, plus no additional servers required. Gonna upload that config shortly!

@sedennial -- would love to hear more about these pages. I can possibly be helpful to you in getting a StoryProvider for them!
